polyphemus/workproc.py

```diff
@@ -82,7 +110,37 @@ def notify(basedir, jobname):
     await sock.close()
 
 
+def valid_stage(stage):
+    """Check if a given string represents a valid stage
+    """
+    if stage not in worker.KNOWN_STAGES.keys():
+        raise argparse.ArgumentTypeError("Unknown stage: %s. Valid stages are: %s" % (stage, KNOWN_STAGES_STR))
+
+    return stage
+
+
 if __name__ == '__main__':
-    p = WorkProc(sys.argv[1] if len(sys.argv) > 1 else INSTANCE_DIR)
-    p.start()
-    p.serve()
+    parser = argparse.ArgumentParser(description='Start Polyphemus Work Processor.')
```

Comment:
```suggestion
    parser = argparse.ArgumentParser(description='Start Polyphemus Work Process.')
```

I used `Proc` because it's a separate process, not because it processes work. :smiley: